docs/source/user_data/create-datasets-for-dashboard.ipynb

```diff
@@ -8,18 +8,31 @@
     "\n",
     "The following notebook steps through how to add a dataset to the MAAP Dashboard.\n",
     "\n",
-    "The MAAP dashboard has 3 environments:\n",
+    "Note, there are 2 scenarios:\n",
+    "\n",
+    "1. Adding a single Cloud-Optimized GeoTIFF (COG) and,\n",
+    "2. Adding multiple spatially distinct COGs as a \"mosaic\" with mosaicJSON.\n",
     "\n",
-    "* dit: biomass.dit.maap-project.org\n",
-    "* staging: biomass.staging.maap-project.org\n",
-    "* production: biomass.maap-project.org\n",
+    "High-level, the steps are:\n",
     "\n",
-    "These instructions will guide you towards adding your dataset to `bimoass.dit.maap-project.org`. The MAAP Dashboard team will \"promote\" changes to staging and production periodically (release schedule forthcoming).\n",
+    "1. Inspect your COG(s) to understand the best rescale and colormap name parameters. Optionally create a mosaic.\n",
+    "2. Define a colormap. Colormaps provide mappings of data values to RGB values.\n",
+    "3. Create a PR to the datasets repo to add or update your dataset.\n",
+    "\n",
+    "The MAAP dashboard has 3 environments:\n",
     "\n",
-    "Pre-requisites:\n",
+    "1. Developer-in-test (DIT): biomass.dit.maap-project.org\n",
+    "2. Staging: biomass.staging.maap-project.org / uat.earthdata.nasa.gov/maap-biomass\n",
+    "3. Production: biomass.staging.maap-project.org / earthdata.nasa.gov/maap-project\n",
```

Comment:
Should we clarify which url is the public facing/share url?

Reply:
```suggestion
    "3. Production: biomass.maap-project.org / earthdata.nasa.gov/maap-biomass\n",
```